### ISSUE TITLE
Sunrise & Sunset do not work

### ISSUE THREAD

**Issue by YarosMallorca** [Neurs12/open-meteo-dart#14]
Minimal repro code:
```dart
import 'package:open_meteo/open_meteo.dart';

void main() async {
  final weather = WeatherApi();
  final response = await weather.request(
    latitude: 52.52,
    longitude: 13.41,
    startDate: DateTime.now(),
    endDate: DateTime.now().add(Duration(days: 1)),
    daily: {WeatherDaily.sunrise},
  );

  print(response.dailyData[WeatherDaily.sunrise]?.values);
}
```

In the console this prints:
```bash
{}
```

**Comment by kemolumi** [comment]
I got it, this is an oversight in the package, it only collects the `double` values type and ignores `int` type that is present in `WeatherDaily.sunrise` for example. I'm going to fix it.

Thanks for reporting this issue!

**Comment by kemolumi** [comment]
This issue now fixed in 2.0.1!

```dart
{
  2024-08-12 05:00:00.000: 1723434297,
  2024-08-13 05:00:00.000: 1723520797
}
```
